app/presenters/export/csv/project_presenter.rb

```diff
@@ -204,4 +204,8 @@ def advisory_board_conditions
 
     @project.advisory_board_conditions
   end
+
+  def completed_grant_payment_certificate_received
+    @project.tasks_data.receive_grant_payment_certificate_date_received if @project.tasks_data.receive_grant_payment_certificate_date_received.present?
```

Comment:
Just s small thing - can we format this in a human readable format, as this will come out as a raw date. Use `to_fs(:govuk)` to format it so it looks like "01 02 2024" or similar

Reply:
Sorry I'm wrong here, was want the `csv` format, not govuk